articles/sharing_settings.md

```diff
@@ -8,23 +8,37 @@ order: 2
+You can use the `Sharing Settings` to grant different levels of access to individuals or `Teams`. Often, users leverage these features to invite people to view or collaborate on a Synapse `Project`. Sharing Settings can apply to an entire project or for individual files or folders within a project.
 
-* Can view
-* Can download
-* Can edit
-* Can edit & delete
-* Administrator
+There are two primary groups: **Public** and **Private**. The `Private` sharing setting limits access to only specified users and teams. By default, new projects are `Private`. The `Public` sharing setting applies to users on the web and any registered, logged-in Synapse user.
 
-`Sharing Settings` can apply to an entire project or for individual files or folders within a project.
+When sharing with specific users and teams, Synapse supports the following permissions: view, download, edit, edit/delete, and administrator.
 
-`Sharing Settings` have two primary groups: **Public** and **Private**. The `Private` sharing setting limits access to only specified users and  teams. By default, new projects are `Private`. The `Public` sharing setting applies to users on the web and any registered, logged-in Synapse user.
+### View permissions
+
+View permissions give a Synapse user the ability to see that something in Synapse exists (like a project, file, or folder). They can discover it using Synapse search, and it will be visible to them if included in a [view](articles/views.html). If there are [annotations](https://docs.synapse.org/articles/annotation_and_query.html) associated with it, they can see these as well. They cannot see the contents of a file, including if a preview of the file is available in the web. View permissions are the only permissions that can be granted to the public (anonymous users).
+
+### Download permissions
+
+Download permissions give a Synapse user the ability to see the contents of a file entity and download the file to their own computer. Having download permissions includes also having view permissions.
+
+### Edit permissions
+
+Edit permissions allows a Synapse user to make changes to something in Synapse. Edit permissions are cumulative with view and download permissions. They can change the name of an entity. They can [change the annotations](/articles/annotation_and_query.html#modifying-annotations) associated with an entity, including removing existing annotations. They can [create a new version](/articles/files_and_versioning.html#uploading-a-new-version) of a file. They can make changes to a [Wiki](/articles/wikis.html). A user can also [change the storage location settings](/articles/custom_storage_location.html) of a Project or Folder. A user with edit permissions cannot delete the entity shared with them.
```

Comment:
This section may do well as a bullet list since most of these sentences start the same with "they can..." or can be changed to start with that.

Reply:
```suggestion
Edit permissions allows a Synapse user to make changes to something in Synapse. Edit permissions are cumulative with view and download permissions. A user with edit permissions can:

- change the name of an entity.
- [change the annotations](/articles/annotation_and_query.html#modifying-annotations) associated with an entity, including removing existing annotations.
- [create a new version](/articles/files_and_versioning.html#uploading-a-new-version) of a file
- make changes to a [Wiki](/articles/wikis.html).
- [change the storage location settings](/articles/custom_storage_location.html) of a Project or Folder.

A user with edit permissions cannot delete the entity shared with them.
```